lib/Controller/Widget.php

```diff
@@ -1720,6 +1752,10 @@ public function saveElements(Request $request, Response $response, $id)
             throw new InvalidArgumentException(__('This Layout is not a Draft, please checkout.'), 'layoutId');
         }
 
+        if ($playlist->isDynamic === 1) {
```

Comment:
Dynamic playlists (as per prior comments can only be created from the Playlist page and therefore `isRegionPlaylist` = false, so we can't get here.

Reply:
Also removed this based on this [commit ](https://github.com/xibosignage/xibo-cms/pull/2711/commits/da53951c8848138bdba4936f28bcd1cf28101c3e) :)